tests/cli/commands/test_task_command.py

```diff
@@ -816,7 +816,7 @@ def test_run_task_with_pool(self):
             session.commit()
 
             assert session.query(TaskInstance).filter_by(pool=pool_name).first() is None
-            task_command.task_run(self.parser.parse_args(self.task_args + ["--pool", pool_name]))
+            task_command.task_run(self.parser.parse_args([*self.task_args, "--pool", pool_name]))
```

Comment:
Maybe we can check whether @uranusjr 's suggestion [here](https://github.com/apache/airflow/pull/33934#discussion_r1311060232) is even better.